### PR TITLE
[docs]: Add info about RAIL model

### PR DESCRIPTION
**Pull request by abrarpathan19**
## Description

Added content for PWA benefits from user point of view. 

## Related Issue

Closes #ISSUE_NUMBER.

### Verification Steps

1. Go to the FOO page.
2. Verify the BAR shows up.
3. Make sure BAZ does a thing.

## Checklist

- [ ] I have added tests to cover my changes, if necessary.
* I have updated the documentation accordingly, if necessary.

### CODE REVIEW

File: pwa-devdocs/src/technologies/overview/index.md [hunk 1]
```diff
@@ -18,6 +18,13 @@ The following features define a basic PWA website:
 -   **Mobile "Install"** - Mobile users can add PWA sites to their home screens and even receive Push notifications from the site.
 -   **Shareable content** - Each page in a PWA site has a unique URL that can be shared with other apps or social media.
 
+This strategy is known as the RAIL model:
```
Comment (jcalcaben) [inline pwa-devdocs/src/technologies/overview/index.md:21]: would be useful to add a link to an external website for more information about this model

**Review by jcalcaben — CHANGES_REQUESTED** [review]

File: pwa-devdocs/src/technologies/overview/index.md (hunk 1 above)
Comment (sirugh) [inline pwa-devdocs/src/technologies/overview/index.md:21]:
```suggestion
This strategy is known as the [RAIL](https://web.dev/rail/) model:
```

**Review by sirugh — CHANGES_REQUESTED** [review]
Please make the requested changes!

**Review by sirugh — APPROVED** [review]
Thanks!

**Review by jcalcaben — APPROVED** [review]

**Pull request merged**